develop/smart-contracts/precompiles.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Advanced Functionalities (Precompiles)
+description: Explores how Polkadot integrates precompiles to run essential functions natively, improving the speed and efficiency of smart contracts on the Hub.
+---
+
+# Advanced Functionalities (Precompiles)
+
+## Introduction
+
+Precompiles play a critical role in enabling high-performance smart contracts by providing native, optimized implementations of frequently used functions. This article explores how Polkadot leverages precompiles within the Revive pallet to enhance efficiency and extend functionality for developers.
```

Comment:
Precompiles are now just for enabling high performance, they also (will) allow contracts to interact with native functionalities like XCM, etc